src/api/app/services/rpmlint_log_extractor.rb

```diff
@@ -8,24 +8,38 @@ def initialize(attributes = {})
     @architecture = attributes[:architecture]
 
     @log_content = ''
+
+    # metrics
+    @rpmlint_log_file_found = true
+    @parse_internal_log_file = false
+    @mark_found = true
   end
 
   def call
-    # Retrieve the content of the 'rpmlint.log' file
-    Backend::Api::BuildResults::Binaries.rpmlint_log(project, package, repository, architecture)
-  rescue Backend::NotFoundError => e
-    # TODO: Remove this `unless` condition once request_show_redesign is rolled out
-    return unless Flipper.enabled?(:request_show_redesign, User.session)
-
-    # Case of removed project, package, repository or architecture:
-    return unless e.summary.include?('rpmlint.log: No such file or directory')
-
-    retrieve_rpmlint_log_from_log
+    time_start = Time.now.to_f
+    content = begin
+      # Retrieve the content of the 'rpmlint.log' file
+      Backend::Api::BuildResults::Binaries.rpmlint_log(project, package, repository, architecture)
+    rescue Backend::NotFoundError => e
+      @rpmlint_log_file_found = false
+
+      # Condition: Flipper.enabled?(...   : TODO: Remove this `unless` condition once request_show_redesign is rolled out
+      # Condition: e.summary.include?(... : Case of removed project, package, repository or architecture
```

Comment:
Are those comments valid?

Reply:
Yes. Sorry, I don't get what do you mean with your question...

Reply:
Yes. Sorry, I don't get what do you mean with your question...

Reply:
Well I don't understand those comments at all. I thought maybe it was something you forgot to remove...

Reply:
Hm...

The comment that starts with `Condition: Flipper.enabled?(...` refers to the first part of the condition.

The comment that starts with `Condition: e.summary.include?(...` refers to the rest of the condition.

I hope this clarifies it.

Reply:
Ah! Now I get it, it's for the line below... Alright!